src/main/java/com/powsybl/openloadflow/ac/equations/AbstractHvdcAcEmulationFlowEquationTerm.java

```diff
@@ -78,6 +81,17 @@ protected double getVscLossMultiplier() {
         return (1 - lossFactor1) * (1 - lossFactor2);
     }
 
+    // FIXME: This method is a copy of HvdcUtils.getHvdcLineLosses() from powsybl-core but it cannot be used yet because this static method is private
+    protected static double getHvdcLineLosses(double rectifierPDc, double nominalV, double r) {
```

Comment:
Thanks, should be fixed soon with https://github.com/powsybl/powsybl-core/pull/3098, that should be included in powsbl-core 6.4.1.

Reply:
You can upgrade the version, powsybl-core 6.4.1 available.

Reply:
Thanks, the code now uses the method getHvdcLineLosses().